syserol/figures/figureS3.py

```diff
@@ -0,0 +1,69 @@
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+import seaborn as sns
+from sklearn.gaussian_process.kernels import ConstantKernel, RBF, WhiteKernel
+from sklearn.gaussian_process import GaussianProcessClassifier
+from syserol.tensor import perform_CMTF
+from syserol.dataImport import load_file
+from syserol.figures.common import getSetup, subplotLabel
+from syserol.classify import class_predictions
+
+
+def makeFigure():
+    tFac, _, _ = perform_CMTF()
+    X = tFac.factors[0]
+
+    outt = class_predictions(X)
+    df = load_file("meta-subjects")
+    y = (df["class.nv"] == "viremic").astype(int)
+    y_pred = outt[2][0]
+
+    kern = ConstantKernel() * RBF(np.ones(2), (1e-2, 1e14))
+    kern += WhiteKernel(noise_level_bounds=(0.001, 0.8))
```

Comment:
This stuff should only be defined in one place. You could make a new function that returns `estG`, then use it here and in `RegressionHelper`?